tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_msu_inventory.yaml

```diff
@@ -35,21 +35,3 @@
             value: NVD_JSON_PATH
         - update_interval:
             value: '5s'
-  - section: sca
```

Comment:
Maybe we should keep these modules turned off, to improve the stability of the test

Reply:
Fixed